docs/data-sources/cloud_provider_aws_cloudwatch_scrape_job.md

```diff
@@ -143,8 +144,10 @@ data "grafana_cloud_provider_aws_cloudwatch_scrape_job" "test" {
 
 - `aws_account_resource_id` (String) The ID assigned by the Grafana Cloud Provider API to an AWS Account resource that should be associated with this CloudWatch Scrape Job.
 - `custom_namespace` (Block List) Zero or more configuration blocks to configure custom namespaces for the CloudWatch Scrape Job to scrape. Each block must have a distinct `name` attribute. When accessing this as an attribute reference, it is a list of objects. (see [below for nested schema](#nestedblock--custom_namespace))
+- `disabled_reason` (String) When the CloudWatch Scrape Job is disabled, this will show the reason that it is in that state.
```

Comment:
For this I don't think we should allow users to pick whatever they like, but rather, use either `disabled_by_user` which we already have, or have a seperate terraform specific value.

https://github.com/grafana/cloud-onboarding/blob/a5593e387fd82fb0586cbaa1aabcd9f8f68a5e05/pkg/hosted-exporters-api/service/exporter.go#L22-L33